Documentation/topics/role-based-access-control.rst

```diff
@@ -82,6 +82,13 @@ command:
 
    $ ovn-sbctl set-connection role=ovn-controller ssl:192.168.0.1:6642
 
+.. note::
+
+   There is currently no pre-defined role for ovn-northd. You must configure
+   a separate listener on the OVN southbound database that ovn-northd can
+   connect to if your deployment topology require ovn-northd to connect to a
+   OVN southbound database instance on a remote machine.
```

Comment:
How about below?


Configuring RBAC for Northd
---------------------------

1. Since there is no rbac support for northd, to access southbound db, make northd listen on a seperate port
say 6648 with a new connection table entry.

      In `machine_3`::

          $ ovn-sbctl -- --id=@conn_uuid create Connection \
             target="ptcp\:6648\:0.0.0.0" \
             -- add  SB_Global . connections=@conn_uuid

Reply:
Other than that Acked-by: Aliasgar Ginwala <aginwala@ebay.com>

Reply:
Yes, I guess we could do both.

The thing that put me off about adding it there is that the tutorial is displaying a single host controller, so it will most likely use a local Unix socket for `ovn-northd` communication.

Having said that, it is worth examplifying how to create the second listener. I'll add the example in the tutorial as you suggested.

How would you feel about port `16442` in the example?

Reply:
@noah8713 I added a sub-section in the tutorial along the lines of what we discussed above.